### PR TITLE
Format component of style_traits

### PR DESCRIPTION
**Pull request by chansuke**
Format `style_traits` with:
```
rustfmt components/style_traits/*.rs
```

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [ ] These changes fix part of #21373.
- [x] These changes do not require tests because they format the components only.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/21620)

### CODE REVIEW

**Comment by jdm** [comment]
```
./components/style_traits/lib.rs:28: extern crate declaration is not in alphabetical order
	expected: webrender_api
	found: servo_arc
```
Could you rearrange the webrender_api crate to be alphabetical?

**Comment by chansuke** [comment]
@jdm Thanks for the feedback.I re-arranged and re-pushed as below.
```
#[cfg(feature = "servo")]
extern crate servo_arc;
extern crate webrender_api;
```

File: components/style_traits/lib.rs [hunk 1]
```diff
@@ -8,27 +8,36 @@
...
-#[macro_use] extern crate malloc_size_of_derive;
+#[macro_use]
+extern crate malloc_size_of_derive;
 extern crate selectors;
-#[cfg(feature = "servo")] #[macro_use] extern crate serde;
-#[cfg(feature = "servo")] extern crate webrender_api;
+#[cfg(feature = "servo")]
+#[macro_use]
+extern crate serde;
+#[cfg(feature = "servo")]
```
Comment (jdm) [inline components/style_traits/lib.rs]: This cfg(feature = "servo") needs to be moved above webrender_api instead of servo_arc.

**Comment by jdm** [comment]
`./mach test-tidy` is still complaining:
```
Checking files for tidiness...
./components/style_traits/lib.rs:30: extern crate declaration is not in alphabetical order
	expected: webrender_api
	found: servo_atoms
```

**Comment by chansuke** [comment]
@jdm fixed again, sorry.

**Comment by jdm** [comment]
@bors-servo retry
* #20734

**Pull request merged**